src/Microsoft.DotNet.Tools.Repl.Csi/Program.cs

```diff
@@ -97,7 +97,7 @@ private static IEnumerable<string> GetRuntimeDependencies(ProjectContext project
         private static string CreateResponseFile(ProjectContext projectContext, string buildConfiguration, string tempOutputDir)
         {
             var outputFileName = projectContext.ProjectFile.Name;
-            var outputFilePath = Path.Combine(tempOutputDir, $"{outputFileName}{Constants.DynamicLibSuffix}");
+            var outputFilePath = Path.Combine(tempOutputDir, $"{outputFileName}.dll");
```

Comment:
It's possible, but I can't think of a good name at this point... Maybe something like `Constants.ManagedBinarySuffix` (it seems the other, `Constants.DynamicLibSuffix` is only valid for native binaries)?